Packs/CVE_2021_40444/ReleaseNotes/1_0_5.md

```diff
@@ -0,0 +1,6 @@
+#### Playbooks
+##### CVE-2021-40444 - MSHTML RCE
+
+- Replaced the **Block Indicators - Generic v2** sub-playbook with the new **Block Indicators - Generic v3** playbook. (Available from Cortex XSOAR 6.5.0).
```

Comment:
Is **Block Indicators - Generic v3** a playbook or a sub-playbook?

Reply:
Playbook